src/Http/Address/GetStreetRequestHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Address;
+
+use CultuurNet\UDB3\Address\StreetSuggester\StreetSuggester;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\QueryParameters;
+use CultuurNet\UDB3\Http\Response\JsonResponse;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+final class GetStreetRequestHandler implements RequestHandlerInterface
+{
+    private StreetSuggester $belgiumStreetSuggester;
+    public function __construct(StreetSuggester $streetSuggester)
```

Comment:
For more clarity it might be good to also call this argument `$belgiumStreetSuggester`? (I like that you made this clear for the property, that it's only used for BE addresses)